airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -56,14 +44,21 @@
 )
 from airflow.providers.cncf.kubernetes.backcompat.pod_runtime_info_env import PodRuntimeInfoEnv
 from airflow.providers.cncf.kubernetes.utils import xcom_sidecar
+from airflow.providers.cncf.kubernetes.utils.pod_manager import PodLaunchFailedException, PodManager, PodPhase
+from airflow.settings import pod_mutation_hook
+from airflow.utils import yaml
+from airflow.utils.context import Context
```

Comment:
Need to add this when `TYPE_CHECKING` otherwise the Kubernetes provider will have a minimum Airflow-version dependency on 2.2.3.

Reply:
Done